taxonomies/ingredients.txt

```diff
@@ -71174,10 +71174,17 @@ fr:Crème dessert saveur vanille
 fr:Crème dessert au chocolat
 # usage:fr:Crème dessert au chocolat: Lait partiellement écrémé 75%, sucre, amidon modifié de maïs, cacao maigre, chocolat 1,9% (pâte de cacao, sucre, émulsifiant : lécithine de tournesol), lactose** et deprotéines _lait,_ épaississants : carraghénanes - gomme xanthane.
 
+<en:pasta
+en:stuffed pasta
+fr:pâte farci, pâtes farcies
```

Comment:
```suggestion
fr:pâtes farcies
```

the singular does not really make sense here.

Reply:
fixed in an upcoming commit